### ISSUE TITLE
Code minimap

### ISSUE THREAD

**Issue by cdelahousse** [SteamedPears/Code-Review#40]
_Imported issue by **[spratt](http://github.com/spratt)** from 2012-06-26T14:42:18Z_
_Originally opened as https://github.com/SteamedPears/CodeReviewClientMaster/issues/7_

---

Sublime text has a "code minimap" at the left side of the screen which shows the code at a very high level.  This is kind of gimicky, but if we highlight the lines with comments then it can help finding comments in large blocks of code.

**Comment by cdelahousse** [comment]
_Imported comment by **[Gankro](http://github.com/Gankro)** from 2012-07-15T05:20:30Z_ *

---

The more I think about this, the less excited I am about it. However if we do this I demand a "comment/diff" heat map indicating activity.

**Comment by cdelahousse** [comment]
_Imported comment by **[spratt](http://github.com/spratt)** from 2012-07-15T05:33:49Z_ *

---

I really like the idea of being able to see at a glance and move directly to wherever there are comments in a big file.  A comment/diff heatmap is a must.

**Comment by spratt** [comment]
Peter had the idea of vertical lines alongside the code indicating comments on the sections of the code next to the lines.

![Code_Review1](https://f.cloud.github.com/assets/80992/128225/683dcc0a-6fb2-11e2-822d-584a7619d09f.png)
![Code_Review2](https://f.cloud.github.com/assets/80992/128226/6841e664-6fb2-11e2-843f-bd5c726f53f2.png)

**Comment by bheesham** [comment]
Yeah, the minimap doesn't seem like such a good idea with the new proposal for the layout.
I do like the idea of the colour-coded lines.

But as someone brought up in the email, what about colour blind people? Unless we consider them an edge case we don't have to worry too much about.
